rsyncwrap/main.py

```diff
@@ -160,7 +189,11 @@ def as_path(self) -> Optional[Path]:
     @cached_property
     def is_path(self) -> bool:
         """Tells us if the line is a path that exists."""
-        return self.as_path and self.as_path.exists()
+        # Does not exist if on a remote. If it's reported by rsync
+        # don't we know that it exists? Do we need to run .exists()?
+        # return self.as_path and self.as_path.exists()
+        # Can we check if it is valid path syntax?
+        return self.as_path
```

Comment:
As you say, rsync already tells us it exists. I don't remember why I even had this method.  I think we can just remove the whole method and use as_path like you're doing already.